partition/androidboot.go

```diff
@@ -75,3 +76,13 @@ func (a *androidboot) SetBootVars(values map[string]string) error {
 	}
 	return env.Save()
 }
+
+func (a *androidboot) Reboot() error {
```

Comment:
I'm telling you to do that! there's a lot of system directories already listed there because of similar reasons, for example  

LocaleDir = filepath.Join(rootdir, "/usr/share/locale")

otoh see my comment about being a bit strange to write over a systemd file